pincer/middleware/ready.py

```diff
@@ -14,15 +14,23 @@
 from ..utils.conversion import construct_client_dict
 
 
-async def on_ready_middleware(self, payload: GatewayDispatch):
-    """
```

Comment:
revert file
- causes conflict with #155 
- may cause issues in library due to changes

Reply:
I don't think the renaming will cause any issues